code/datums/datum_click_handlers.dm

```diff
@@ -103,7 +103,12 @@
 /datum/click_handler/fullauto/MouseDown(object,location,control,params)
 	if(!isturf(owner.mob.loc)) // This stops from firing full auto weapons inside closets or in /obj/effect/dummy/chameleon chameleon projector
 		return FALSE
-	
+	if((owner.mob.Adjacent(location) && !(owner.mob.a_intent == "harm"))||owner.mob.in_throw_mode)//Occulus Edit Start
```

Comment:
Could make these bools easier to read (as well as tossing in a short circuit) via:
```suggestion
	if(owner.mob.in_throw_mode || (owner.mob.Adjacent(location) && owner.mob.a_intent != "harm"))//Occulus Edit Start
```